pyiceberg/table/__init__.py

```diff
@@ -1229,7 +1240,8 @@ def with_case_sensitive(self: S, case_sensitive: bool = True) -> S:
 
 
 class ScanTask(ABC):
-    pass
+    def size_in_bytes(self) -> int:
```

Comment:
would it be better to mark this method as abstract?

Reply:
sure, marked as abstract